### PR TITLE
[CARBONDATA-808] Added Example for Presto

### PR DESCRIPTION
**Pull request by PallaviSingh1992**

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1625/

File: examples/presto/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+## Pre-requisites to execute the Presto Example
```
Comment (chenliang613) [inline examples/presto/README.md]: please put PrestoExample to integration/presto, currently all related files of presto be put to integration/presto, once presto integration module be finished completely, then we will move to corresponding folder.

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1945/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1947/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1948/

**Comment by chenliang613** [comment]
@PallaviSingh1992 please rebase this pr based on master

**Comment by PallaviSingh1992** [comment]
Updating the example after Presto branch merged into master.

**Pull request closed without merging**